src/main/java/roomescape/domain/Repository/JdbcReservationRepository.java

```diff
@@ -0,0 +1,56 @@
+package roomescape.domain.Repository;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.stereotype.Repository;
+import roomescape.domain.Model.Reservation;
+import roomescape.domain.exception.NotFoundReservationException;
+
+import java.sql.PreparedStatement;
+import java.time.LocalDateTime;
+import java.util.List;
+
+@Repository
+public class JdbcReservationRepository implements BasicRepository{
+    private final JdbcTemplate jdbcTemplate;
+
+    public JdbcReservationRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    @Override
+    public List<Reservation> findAll() {
+        String sql = "select * from reservation";
```

Comment:
```suggestion
        String sql = "SELECT * FROM reservation";

```
다른 코드들처럼 대문자로 통일하면 보기 편할 거 같아요!